snapcraft/internal/parser.py

```diff
@@ -233,6 +233,22 @@ def _process_wiki_entry(entry, master_parts_list):
         master_parts_list.update(parts_list)
 
 
+def _handle_wiki_entry(entry, master_parts_list):
+    wiki_errors = 0
+    try:
+        _process_wiki_entry(entry, master_parts_list)
+    except InvalidEntryError as e:
+        logger.warning('Invalid wiki entry: {!r}'.format(e))
+        wiki_errors = 1
+    except sources.MissingPackageError as e:
+        logger.warning(e)
+        # TODO: support multiple types of errors, see
+        # https://bugs.launchpad.net/snapcraft/+bug/1616613
+        wiki_errors = 1
+
+    return wiki_errors
```

Comment:
Yes

On Sep 14, 2016 13:54, "Sergio Schvezov" <notifications@github.com> wrote:

> In snapcraft/internal/parser.py
> <https://github.com/snapcore/snapcraft/pull/742#discussion_r78800815>:
>
> > @@ -233,6 +233,22 @@ def _process_wiki_entry(entry, master_parts_list):
> >          master_parts_list.update(parts_list)
> >
> >
> > +def _handle_wiki_entry(entry, master_parts_list):
> > +    wiki_errors = 0
> > +    try:
> > +        _process_wiki_entry(entry, master_parts_list)
> > +    except InvalidEntryError as e:
> > +        logger.warning('Invalid wiki entry: {!r}'.format(e))
> > +        wiki_errors = 1
> > +    except sources.MissingPackageError as e:
> > +        logger.warning(e)
> > +        # TODO: support multiple types of errors, see
> > +        # https://bugs.launchpad.net/snapcraft/+bug/1616613
> > +        wiki_errors = 1
> > +
> > +    return wiki_errors
>
> @josepht <https://github.com/josepht> are you going to take an action
> about this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/snapcore/snapcraft/pull/742/files/847aa5466bf9cdf6e9bee5c0b8a01f06a8a0444c#r78800815>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnbyD6Dp8q8ZtRG2_qoIJoq6JMFpLU1ks5qqDTigaJpZM4Jn4qH>
> .
>